_test/shared/matlab_xunit/xunit/TestSuite.m

```diff
@@ -79,14 +79,17 @@
             if nargin < 2
                 monitor = CommandWindowTestRunDisplay();
             end
+            if nargin<3
+                num_tests_runed = 0;
```

Comment:
`num_tests_run`

Reply:
would not spend time on this, but fine, changed and merging without test